### ISSUE TITLE
Could you please analyze why MS-GCNet is much better than MS-PSMNet?

### ISSUE THREAD

**Issue by SpadeLiu** [ccj5351/MS-Nets#2]
As in the experiments, MS-GCNet is much better than MS-PSMNet. But the parameters for cost aggregation of them are close, 1.9M vs 2.3M? What is the reason? Is it because the cost volume of GCNet is in size H/2 * W/2 and contains more details?

**Comment by ccj5351** [comment]
Yes. First, I agree with it is partially due to the differences in cost volume resolutions, i.e., 1/4 in PSMNet VS 1/2 in GCNet.  Second, I think it is more due to the fact that the 3D encoder-decoder architectures themselves are different, even though with an equivalent amount of parameters.

**Comment by SpadeLiu** [comment]
Thanks for the reply. Haven't you tried GANet? It utilizes image information during cost aggregation and it naturally generalizes better.

**Comment by ccj5351** [comment]
I agree with you. For this project, we did not try the GANet backbone. We use GANet for some other projects.
